usl_pipeline/cloud_functions/main.py

```diff
@@ -574,10 +582,6 @@ def _build_feature_matrix_from_archive(
                 green_areas = _read_polygons_from_byte_stream(fd)
             elif name == file_names.SOIL_CLASSES_TXT:
                 soil_classes = _read_polygons_from_byte_stream(fd)
-            # Handle heat model input files (WPS)
-            # TODO: Refactor to also handle un-tarred files
-            elif re.search(file_names.WPS_DOMAIN3_NC_REGEX, name):
```

Comment:
Can we keep this in and support both files in and out of the tar?

Reply:
sure thing - added back in